microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md

```diff
@@ -21,53 +21,79 @@ ms.prod: m365-security
 ms.date: 06/10/2021
 ---
 
-# Review and manage remediation actions in Office 365
-
 **Applies to**
 - [Microsoft Defender for Office 365 plan 2](defender-for-office-365.md)
-- [Microsoft 365 Defender](../defender/microsoft-365-defender.md)
+
+# Review and manage remediation actions in Office 365
 
 As automated investigations on email & collaboration content result in verdicts, such as *Malicious* or *Suspicious*, certain remediation actions are created. In Microsoft Defender for Office 365, remediation actions can include:
 
-- Blocking a URL (time-of-click)
 - Soft deleting email messages or clusters
-- Quarantining email or email attachments
 - Turning off external mail forwarding
 
-These remediation actions are not taken unless and until your security operations team approves them. We recommend reviewing and approving any pending actions as soon as possible so that your automated investigations complete in a timely manner. In some cases, you can undo a remediation action.
+These remediation actions are not taken unless and until your security operations team approves them. We recommend reviewing and approving any pending actions as soon as possible so that your automated investigations complete in a timely manner. In some cases, you can re-consider submitted actions.
```

Comment:
```suggestion
These remediation actions are not taken unless and until your security operations team approves them. We recommend reviewing and approving any pending actions as soon as possible so that your automated investigations complete in a timely manner. In some cases, you can reconsider submitted actions.
```